book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def rent_book(date, person)
+    Rental.new(date, person, self)
```

Comment:
- [ ] Kindly fix the order of your arguments. The `#initialize` method in the Rental class has the parameter in this arrangement: date, book, person - so this should be:

  ```
  Rental.new(date, self, person)
  ```